commands/cli.go

```diff
@@ -27,12 +28,30 @@ func CreateCli(version string) *cli.App {
 			Name:  "exclude-region",
 			Usage: "regions to exclude",
 		},
+		cli.StringFlag{
+			Name:  "older-than",
+			Usage: "duration of resources' creation date to delete from",
+			Value: "0s",
+		},
 	}
-	app.Action = errors.WithPanicHandling(awsNuke)
 
+	app.Action = errors.WithPanicHandling(awsNuke)
 	return app
 }
 
+func parseDurationParam(paramValue string) (*time.Time, error) {
+	duration, err := time.ParseDuration(paramValue)
+	if err != nil {
+		return nil, errors.WithStackTrace(err)
+	}
+
+	// make it negative so it goes back in time
+	duration = -1 * duration
+
+	excludeAfter := time.Now().Add(duration)
```

Comment:
No need to use negatives. Just do `time.Now().Sub(...)`.

Reply:
The thing is `Sub` is used to subtract a time from another and return a duration. I'd still have to do some sort of preparation and won't be able to use `duration` directly

Reply:
Oh, how weird that `Add` and `Sub` aren't symmetric! OK, carry on then :)